client/my-sites/hosting/cache-card/index.js

```diff
@@ -156,32 +120,35 @@ export const CacheCard = ( {
 			<CardHeading id="cache" size={ 20 }>
 				{ translate( 'Cache' ) }
 			</CardHeading>
-			<CardBody>
-				<EdgeCacheDescription>
+			<div className="flex flex-col">
```

Comment:
I know that the changes to the Hosting Cache card is just a demo how Tailwind could be used, but to make it mergeable into production, the changes shouldn't be inconsistent like here: using `<Card>` and `<CardHeading>`, but not `<CardBody>`. Using `ToggleControl`, but not `ToggleContainer`.

Reply:
`Card` is a component we already have. 
In this specific component, we used a div to represent a `CardBody`.We just named it like that because of the styled-components. It was just a naming convention.
Same goes with `ToggleContainer`. I can create a functional component, just to give the naming convention back, but is it really necessary?

What I did was to remove `styled`, everything is the same as it would be without it.